pilot/cmd/pilot-agent/status/util/listeners.go

```diff
@@ -31,28 +32,24 @@ var (
 
 // GetInboundListeningPorts returns a map of inbound ports for which Envoy has active listeners.
 func GetInboundListeningPorts(localHostAddr string, adminPort uint16, nodeType model.NodeType) (map[uint16]bool, string, error) {
-	buf, err := doHTTPGet(fmt.Sprintf("http://%s:%d/listeners", localHostAddr, adminPort))
+	buf, err := doHTTPGet(fmt.Sprintf("http://%s:%d/listeners?format=json", localHostAddr, adminPort))
```

Comment:
a bit annoying they changed the format on us, but doing this as json seems much better